packages/components/cascader-panel/index.ts

```diff
@@ -3,7 +3,7 @@ import type { App } from 'vue'
 import type { SFCWithInstall } from '@element-plus/utils'
 
 CascaderPanel.install = (app: App): void => {
-  app.component(CascaderPanel.name, CascaderPanel)
+  app.component(CascaderPanel.name as string, CascaderPanel)
```

Comment:
Maybe just use the non-null assertion operator.
```suggestion
  app.component(CascaderPanel.name!, CascaderPanel)
```